README.md

```diff
@@ -34,11 +34,19 @@ For installation using conda, paste the following code into the environment conf
 In order to access the colours in the package, use the command:
 
 ```python
-from GOV_UK_Colours.colours import GovUKColors
+from gov_uk_dashboards.colours.colours import GovUKColours
```

Comment:
This namespace feels quite excessive, is there a way to avoid the duplicate `.colours.colours` part?